app/helpers.php

```diff
@@ -137,3 +137,10 @@ function has_include($request, $include)
 
     return false;
 }
+
+function formatDateAttribute($value, $timezone)
```

Comment:
It might also be nice to set a default `$timezone` value here – perhaps `$timezone = 'UTC'`?